x-pack/plugins/security_solution/public/flyout/right/tabs/table_tab.tsx

```diff
@@ -5,16 +5,44 @@
  * 2.0.
  */
 
-import { EuiText } from '@elastic/eui';
 import type { FC } from 'react';
 import React, { memo } from 'react';
-import { TABLE_TAB_CONTENT_TEST_ID } from './test_ids';
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { ERROR_MESSAGE, ERROR_TITLE } from './translations';
+import { TimelineTabs } from '../../../../common/types';
+import { EventFieldsBrowser } from '../../../common/components/event_details/event_fields_browser';
+import { useRightPanelContext } from '../context';
+import { TABLE_TAB_ERROR_TEST_ID } from './test_ids';
 
 /**
- * Table view displayed in the document details expandable flyout right section
+ * Table view displayed in the alert details expandable flyout right section
```

Comment:
I think you were fine with `document` here as it would show for a regular event as well

Reply:
i didn't want to make this change, thanks for catching this!